### ISSUE TITLE
Mobility hindrance consequences: list complete?

### ISSUE THREAD

**Issue by berdienderoo** [Informatievlaanderen/GIPOD#23]
Some months ago, a mapping for hindrance was available on the confluence space. This page seems no longer available. Will this be added in the future?

Is the list of mobility hindrance consequence types that can be requested through the api (taxonomies) complete? If so, how will e.g. 'pedestrians on the roadway' be mapped to GIPOD²?

In the taxonomies, I also see a list of mobility hindrance locations. I don't see at first glance where this code list is used in the api? I don't see it in the 'register mobility hindrance' request.

**Comment by naomioste** [comment]
Hi Berdien, 

thanks for your questions.

- This page is being adapted for the moment, since we changed some mappings. It will be published once it is finished.
- The list is complete. "Pedestrians on the roadway" will no longer be available, so it will not be mapped either. The members of the Business Workgroup decided that this is of no extra value for the hindrance in GIPOD. (Of course, there can still be such a sign in the street)
- The hindrance locations won't be used in GIPOD² anymore, since the information in the hindrance consequences is more specific and accurate. So it is no longer needed to map the locations as well.

**Comment by berdienderoo** [comment]
Hi Naomi,

Thanks for the answer. 
So, the taxonomy 'Mobility hindrance locations' will no longer be used (API call '/api/v1/taxonomies/mobility-hindrance/locations')?

**Comment by doomsju** [comment]
Correct, this endpoint will be removed from the API.

**Comment by berdienderoo** [comment]
ok, thanks for the quick update.
